msteams-platform/messaging-extensions/api-based-overview.md

```diff
@@ -8,79 +8,574 @@ ms.author: anclear
 ms.date: 10/19/2023
 ---
 
-# Build message extensions using API
+# API-based message extension
 
 > [!NOTE]
 > API-based message extensions only support search commands.
 
-Message extensions built using API (API-based) use a web service to manage user requests and responses and don't require a bot registration. You can configure and deploy API-based message extensions using Developer Portal for Teams and Teams Toolkit for Visual Studio Code, command line interface (CLI), or Visual Studio. API-based message extensions help your apps to interact directly with third-party data, apps, and services, enhancing its capabilities. With API-based message extension, you can:
+Message extensions built using API (API-based) use a web service to manage user requests and responses and don't require a bot registration. API-based message extensions are a Microsoft Teams app capability that integrates external APIs directly into Teams, enhancing your app's usability and offering a seamless user experience. API-based message extensions support search commands and can be used to fetch and display data from external services within Teams, streamlining workflows by reducing the need to switch between applications. API-based message extensions help your apps to interact directly with third-party data, apps, and services, enhancing its capabilities. With API-based message extension, you can:
```

Comment:
This looks different from the doc, intentional?

![image](https://github.com/user-attachments/assets/ee49152b-5ada-4a85-9db0-06548b7267d5)

Reply:
Hi @heyitsaamir,

Yes It's updated to be in line with Microsoft Writing Style.